docs/pages/deploy-a-cluster/helm-deployments/aws.mdx

```diff
@@ -315,16 +314,39 @@ aws:
   dynamoAutoScaling: false                        # Whether Teleport should configure DynamoDB's autoscaling.
 highAvailability:
   replicaCount: 2                                 # Number of replicas to configure
+ingress:
+  enabled: true
+  spec:
+    ingressClassName: alb
 annotations:
-  service:
+  ingress:
+    alb.ingress.kubernetes.io/target-type: ip
+    alb.ingress.kubernetes.io/backend-protocol: HTTPS
+    alb.ingress.kubernetes.io/scheme: internet-facing
+    alb.ingress.kubernetes.io/load-balancer-attributes: idle_timeout.timeout_seconds=350
+    alb.ingress.kubernetes.io/healthcheck-protocol: HTTPS
+    alb.ingress.kubernetes.io/success-codes: 200,301,302
```

Comment:
It's weird; I've always kinda preferred that we _don't_ invisibly add annotations into chart defaults, as I think explicit behaviour is better than implicit. In hindsight, I now feel like the automation I previously did for ACM (where we automatically override the backend protocol in certain situations: https://github.com/gravitational/teleport/blob/master/examples/chart/teleport-cluster/templates/proxy/service.yaml#L2-L20) wasn't at all obvious and several people who tried to use ACM without DynamoDB/S3 (i.e. they used `chartMode: standalone` rather than `chartMode: aws`) didn't have any idea that there were extra annotations which would need to be set for things to work.

Also, with the requirement for the AWS LB controller I feel like anyone Googling these annotations is more likely to find out that the in-tree controller doesn't support them.

I do agree that it's kinda messy as-is, but I also like that it's clearly showing people exactly what they're doing and they're likely to have a tighter grasp of what's actually going on inside their cluster.